test/test-applications/integrations/aspnet/Samples.AspNet/packages.config

```diff
@@ -11,10 +11,10 @@
   <package id="OpenTelemetry.Api" version="1.2.0-beta1" targetFramework="net461" />
```

Comment:
Consider to update to rc1 verwsion. IT will be in sync with other OpenTelemetry stuff.